<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewMedium</class>
 <widget class="QDialog" name="AddNewMedium">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>add new medium</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="typeMediumLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonBook">
          <property name="text">
           <string>Book</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonCD">
          <property name="text">
           <string>&amp;CD</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonDVD">
          <property name="text">
           <string>DVD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>title</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="titleEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
                <property name="placeholderText">
                 <string>War and Peace</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>author</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="authorEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
                <property name="placeholderText">
                 <string>Tolstoy</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>year published</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="yearEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumDateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>1753</year>
                  <month>9</month>
                  <day>14</day>
                 </datetime>
                </property>
                <property name="maximumDate">
                 <date>
                  <year>2019</year>
                  <month>12</month>
                  <day>31</day>
                 </date>
                </property>
                <property name="displayFormat">
                 <string>yyyy</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>number of pages</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="pagesEdit">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>2000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>duration</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="durationEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <double>300.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>3.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>video quality</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="qualityEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>240</number>
                </property>
                <property name="maximum">
                 <number>1080</number>
                </property>
                <property name="singleStep">
                 <number>120</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="addButton">
                <property name="text">
                 <string>add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cancelButton">
                <property name="text">
                 <string>cancel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
